spec/automation/unit/method_validation/vm_migrate_task_complete_spec.rb

```diff
@@ -11,11 +11,21 @@
     expect(GenericMailer).to receive(:deliver).with(:automation_notification,
                                                     hash_including(:to   => user.email,
                                                                    :from => "evmadmin@example.com"
-                                                                  )
+                                                                )
```

Comment:
@billfitzgerald0120 The change in whitespace here is revealing the rubocop warnings around alignment.  Please update the closing parentheses, like you did in the new test below, to remove the warnings.